content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -157,116 +191,293 @@ Automatic reordering rules, enabled by setting the reordering rule's :guilabel:`
+      :guilabel:`Forecast` and :guilabel:`To Order` quantities are zero.
 
-   .. image:: reordering_rules/visibility-days.png
-      :align: center
-      :alt: Show the replenishment dashboard with the correct and incorrect visibility days set.
+   .. figure:: report/five-forecast.png
+      :alt: Show forecasted report.
 
-.. _inventory/product_management/route:
+      Opening the :guilabel:`Forecasted Report` shows the :guilabel:`Forecasted` units is `5.00`.
 
-Preferred route
+.. _inventory/product_management/visibility-days:
+
+Visibility days
 ===============
 
-Odoo allows for multiple routes to be selected under the :guilabel:`Inventory` tab on each product
-form. For instance, it is possible to select both :guilabel:`Buy` and :guilabel:`Manufacture`, thus
-enabling the functionality of both routes.
+*Visibility days* enables the ability to determine if additional quantities should be added to the
+planned replenishment. Odoo checks:
 
-Odoo also enables users to set a preferred route for a product's reordering rule. This is the route
-that the rule defaults to if multiple are selected. To select a preferred route, begin by navigating
-to :menuselection:`Inventory app --> Configuration --> Reordering Rules` or
-:menuselection:`Inventory app --> Operations --> Replenishment`.
+- If forecasted stock on the forecasted date will drop below the minimum in the reordering rule.
+- **Only if** it is time to reorder, visibility days check additional future demand, by the
+  specified number of days.
 
-Click inside of the column on the row of a reordering rule, and a drop-down menu shows all available
-routes for that rule. Select one to set it as the preferred route.
+This feature helps consolidate orders by grouping immediate and near-future needs, reducing
+transport costs and enabling supplier discounts for larger orders.
 
-.. image:: reordering_rules/select-preferred-route.png
-   :align: center
-   :alt: Select a preferred route from the drop-down.
+To set visibility days to incorporate orders for a specified number of days in the future, navigate
+to :menuselection:`Inventory app --> Operations --> Replenishment`, or by clicking the *Reordering
+Rules* smart button from the product form.
+
+Next, enable the :guilabel:`Visibility Days` field by clicking the :icon:`oi-settings-adjust`
+:guilabel:`(adjust)` icon to the far right and choosing the feature from the drop-down menu. Then,
+enter the desired visibility days.
 
 .. important::
-   If multiple routes are enabled for a product but no preferred route is set for its reordering
-   rule, the product is reordered using the selected route that is listed first on the
-   :guilabel:`Inventory` tab of the product form.
+   The forecasted date is never pushed forward or extended; only if the stock at the forecasted date
+   arrives below the minimum threshold is when Odoo checks the extra visibility days.
+
+Example where visibility days is triggered
```

Comment:
This example is SO helpful! Great job!! Do you think it should be in an `.. example::` block instead?